app/routes/repos.py

```
@@ -4,189 +4,192 @@
+					return APIResponse.success(
+						message="Repositories retrieved successfully",
+						data={"total_count": total_count, "repos": repos},
+					)
+				else:
+					return APIResponse.error(
+						message=f"Unsupported Git hosting provider: {hosting}",
+						status_code=status.HTTP_400_BAD_REQUEST,
+					)
+			except Exception:
+				return APIResponse.error(
+					message=f"Failed to fetch repositories: {str(e)}",
```

Ruff (0.11.9)

Undefined name `e`

(F821)